airbyte/constants.py

```diff
@@ -65,6 +65,19 @@ def _str_to_bool(value: str) -> bool:
     return bool(value) and value.lower() not in {"", "0", "false", "f", "no", "n", "off"}
 
 
+OVERRIDE_TEMP_DIR: Path | None = (
```

Comment:
Nit: If we could rename to `TEMP_DIR_OVERRIDE`, it will sort nicely in auto-complete with `TEMP_FILE_CLEANUP` below.

Wdyt?